lightwood/api/json_ai.py

```diff
@@ -890,7 +870,7 @@ def code_from_json_ai(json_ai: JsonAI) -> str:
 # Fit Models
 # --------------- #
 # Assign list of mixers
-self.mixers = [{', '.join([call(x) for x in list(json_ai.outputs.values())[0].mixers])}]
+self.mixers = [{', '.join([call(x) for x in json_ai.model["args"]["submodels"]])}]
```

Comment:
This jumps to me a bit in terms of consistency. What is the difference between `mixers` and `submodels`? I guess from this PR we call `submodel` an "unrealized" `mixer`.

It may be better to opt for either `mixer` or `submodel` and use it all the way through to the autogenerated code? I'm not too convinced either way, but I can see how this could generate confusion for users.

Reply:
I agree, I was too lazy to do the change but I shouldn't have been... implementing now (with all incoming bugs :p)... sigh

Reply:
Ok... looking at this, no no no, 278 occurrences, loads of bugs, I will make an issue to rename and get to it later.

Reply:
https://github.com/mindsdb/lightwood/issues/829